lib/govuk_connect/cli.rb

```diff
@@ -73,9 +75,11 @@ def bold(string)
     "app-console" => "Launch a console for an application.  For example, a rails console when connecting to a Rails application.",
     "app-dbconsole" => "Launch a console for the database for an application.",
     "rabbitmq" => "Setup port forwarding to the RabbitMQ admin interface.",
```

Comment:
Do you plan to remove this at a later date?

Reply:
Yes, we'll need to support both until the existing rabbitmq EC2 instances are destroyed (that'll be the last part of my work) at which point I'll remove the "rabbitmq" option